### PR TITLE
Remove the super admin of state.

### PR DESCRIPTION
**Pull request by kaikai1024**
### Description of the Change

super admin account is no need for state.

### Alternate Design

NO

### Benefits

Keep the state clean.

### Possible Drawbacks

call the amend address using solidity assembly.

### Verification Process

Unit test

### Applicable Issues

None

### CODE REVIEW

File: cita-executor/core/src/tests/amend_data_test.rs [hunk 1]
```diff
@@ -66,53 +108,38 @@ fn call_vm(
...
     data.append(&mut key.to_vec());
+    // Get value from key use transact interface
+    // `value=4` means the operation of getting value from key.
+    let result = transact(&mut state, &data, U256::from(4), true);
+    assert!(result.is_ok());
+    // TODO Add output of transact in executive.rs when amend get-value
+    // println!("result: {:?}", result.clone().unwrap());
+
+    // Get value from key use call interface
+    let address: Address = reserved_addresses::AMEND_ADDRESS.into();
```
Comment (ouwenkg) [inline cita-executor/core/src/tests/amend_data_test.rs]: It's better to slipt this test into two cases `superAdmin` or `Not superAdmin`

Comment (kaikai1024) [reply cita-executor/core/src/tests/amend_data_test.rs]: It just fix the older test. so if follow the old style.

There are othere problems beside what you say:

* the other amend operation
* the output(check the todo), maybe it's the problem of progress of executive.

**Comment by kaikai1024** [comment]
Already check the super admin:

https://github.com/cryptape/cita/blob/develop/cita-executor/core/src/executive.rs#L342-L350

**Comment by ouwenkg** [comment]
Please update the branch you know.

**Comment by kaikai1024** [comment]
DONE

**Pull request merged**